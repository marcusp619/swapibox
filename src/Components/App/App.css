/*------------------------------------*\
  #APP
\*------------------------------------*/

.App {

}

.Main {
  display: grid;
  grid-template-columns: 25% 1fr;
}

.Aside {
  border: 2px solid blue;
  background: grey;
  height: 100vh;
}

.Button--container {
  display: flex;
  flex-direction: row;
  justify-content: center;
  border: 1px solid black;
  margin: 10px;
}

.Button--container button {
  height: 50px;
  margin-right: 5px;
}
