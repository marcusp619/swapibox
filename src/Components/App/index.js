import React, { Component } from 'react';
import './App.css';
import Header from '../Header/';
import Button from '../Button/';
import Movie from  '../Movie/';

class App extends Component {
  constructor() {
    super();
  }

  render() {
    return (
      <div className="App">
        <Header />
        <main className="Main">
          <div className="Aside">
            <Movie />
          </div>
          <div className="Button--container">
            <Button title={'People'} />
            <Button title={'Planets'}/>
            <Button title={'Vehicles'}/>
          </div>
          <div className="CardContainer">
          </div>   
        </main>
      </div>
    );
  }
}

export default App;
