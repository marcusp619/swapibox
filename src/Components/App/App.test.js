import React from 'react';
import ReactDOM from 'react-dom';
import App from './App';
const enzyme = require('enzyme');

it('renders without crashing', () => {
  const wrapper = enzyme.shallow(<App />);

   expect(wrapper).toMatchSnapshot();
});
