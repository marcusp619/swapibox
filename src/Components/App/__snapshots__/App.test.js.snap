// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders without crashing 1`] = `
<div
  className="App"
>
  <Header />
  <div
    className="Button--container"
  >
    <Button />
    <Button />
    <Button />
  </div>
  <h1>
    SWAPI BOX
  </h1>
</div>
`;
